src/App.js

```diff
@@ -1,26 +1,40 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css'
+import './App.css'
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import SignIn from './components/signIn'
+import SignUp from './components/signUp'
+
+function App () {
+  return (<Router>
+    <div className='App'>
+      <nav className='navbar navbar-expand-lg navbar-light fixed-top'>
+        <div className='container'>
+          <div className='collapse navbar-collapse' id='navbarTogglerDemo02'>
+            <ul className='navbar-nav ml-auto'>
+              <li className='nav-item'>
+                <Link className='nav-link' to='/auth/signin'>Sign In</Link>
+              </li>
+              <li className='nav-item'>
+                <Link className='nav-link' to='/auth/signup'>Sign up</Link>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
```

Comment:
There's too much nesting here. I'd recommend dividing this into a few components:
- `NavBar`: It would contain the styles for the navigation bar and the list of elements.
- `NavBarItem`: It represents each item in the navigation bar. This way you can reuse the code and avoid duplication (_**DRY**: Don't Repeat Yourself_). This component could receive you props: `children` (for the text), `to` (for the `to` prop of the `Link` component).